source/renderer/app/components/wallet/WalletSendForm.js

```diff
@@ -200,7 +199,10 @@ export default class WalletSendForm extends Component<Props, State> {
     }
   };
 
-  handleSubmitOnEnter = submitOnEnter.bind(this, this.handleOnSubmit);
+  handleSubmitOnEnter = (event: KeyboardEvent): void => {
```

Comment:
@danielmain wouldn't it be better to rewrite the `submitOnEnter` method from https://github.com/input-output-hk/daedalus/blob/develop/source/renderer/app/utils/form.js#L1 ?

Reply:
As agreed, we don't change `submitOnEnter` since it is already working in other components

Reply:
Deal!